CMakeLists.txt

```diff
@@ -31,9 +33,7 @@ IF(CMAKE_BUILD_TYPE MATCHES [cC][oO][vV][eE][rR][aA][gG][eE])
 ENDIF(CMAKE_BUILD_TYPE MATCHES [cC][oO][vV][eE][rR][aA][gG][eE])
 
 find_package(LibPhoneNumber REQUIRED)
-find_package(Qt5Core)
-find_package(Qt5DBus)
-find_package(Qt5Network)
+find_package(Qt5 REQUIRED COMPONENTS Core DBus Network Test)
```

Comment:
I thnk `Qt5Test` should not be strictly required, it should be optional. Should be required only in `tests` subdirectory (add `find_package` there).

Reply:
Sure, please note however that the test are being build unconditionally at the moment. I moved the `find_package ... test)` nevertheless